screeninfo/screeninfo.py

```diff
@@ -13,23 +13,22 @@
 }
 
 
-def _get_monitors(enumerator: Enumerator) -> T.List[Monitor]:
-    return list(ENUMERATOR_MAP[enumerator].enumerate_monitors())
-
-
 def get_monitors(
     name: T.Union[Enumerator, str, None] = None
 ) -> T.List[Monitor]:
     """Returns a list of :class:`Monitor` objects based on active monitors."""
     enumerator = Enumerator(name) if name is not None else None
 
     if enumerator is not None:
-        return _get_monitors(enumerator)
+        return list(ENUMERATOR_MAP[enumerator].enumerate_monitors())
```

Comment:
Possible refactor:

```suggestion
    if name is not None:
        return list(ENUMERATOR_MAP[Enumerator(name)].enumerate_monitors())
```

Reply:
Thanks. Fixed. I'll hold off with merging this until we land #60